docs/ref/AbsoluteVote.md

```diff
@@ -0,0 +1,217 @@
+
+# *contract* AbsoluteVote is  
+
+
+
+## Functions
+
+
+###  setParameters
+
```

Comment:
The format should include also the function and param description (not just type).
It also will be nice to see the functions full signature (modifiers public/private/pure..and so on..).